ioet_feature_flag/helpers/validators.py

```diff
@@ -0,0 +1,36 @@
+
+_INVALID_FUNCTION_CODES = [
+    (lambda: True).__code__.co_code,
+    (lambda: False).__code__.co_code,
+]
+
+
+def are_args_boolean(*args) -> bool:
+    for arg in args:
+        if isinstance(arg, bool):
+            return True
+    return False
+
+
+def are_args_different_types(*args) -> bool:
+    if not args:
+        return False
+    last_type = str(type(args[0]))
```

Comment:
This should be the first type?

Reply:
hmm good question, we could also initialize this variable as `None`, what do you think?

Reply:
So the idea is that this variable actually holds the last type? ie, the type of the last `arg` in the array?

Reply:
oh wait, I didn't realize that this function isn't actually complete lol, but my idea was to compare the current argument against the previous one, I gotta fix that